### PR TITLE
Add javascript template

### PR DESCRIPTION
**Pull request by elbywan**

### CODE REVIEW

**Comment by elbywan** [comment]
Forgot prelude / postlude, adding now…

**Comment by elbywan** [comment]
Done 🙌

File: templates/javascript/README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Generated Snips skill
+
+This is a generated javascript snips skill using the `snips-template` tool.
```
Comment (anthonyray) [inline templates/javascript/README.md]: let's call it an app ?

File: templates/javascript/README.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+It is compatible with the format expected by the `snips-skill-server`
+
+## Setup
+
+This skill requires some javascript dependencies to work properly, these are
```
Comment (anthonyray) [inline templates/javascript/README.md]: same

File: templates/javascript/README.md (hunk 1 above)
Comment (elbywan) [reply templates/javascript/README.md]: To be honest I just copy pasted the python description with some renaming. 😉

**Comment by anthonyray** [comment]
Let's merge this one and #17

**Pull request merged**